locale/sw.yml

```diff
@@ -1405,7 +1405,7 @@ class: "darasa"
 def: "fanya"
 defined?: "ilidefiniwa?"
```

Comment:
I don't think there's something like ilidefiniwa

Reply:
imefafanuliwa makes more sense

Reply:
yeah that would be more acceptable